dynawo/sources/Models/Modelica/Dynawo/Examples/InertialGrid/StaticCase.mo

```diff
@@ -0,0 +1,103 @@
+    Placement(visible = true, transformation(origin = {16, -26}, extent = {{-10, -10}, {10, 10}}, rotation = 90)));
+  Dynawo.Electrical.Lines.Line line3(BPu = 0, GPu = 0, RPu = R3Pu * L3, XPu = X3Pu * L3) annotation(
+    Placement(visible = true, transformation(origin = {50, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 180)));
+  Dynawo.Electrical.Buses.InfiniteBus slackBus(UPhase = 0, UPu = 1) annotation(
+    Placement(visible = true, transformation(origin = {-36, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 90)));
+  Dynawo.Electrical.Machines.Simplified.GeneratorFictitious Gen2(Alpha = 0, Beta = 0, PGen0Pu = 3.3, QGen0Pu = 0, U0Pu = 1, i0Pu = ComplexMath.conj(Complex(Gen2.PGen0Pu, Gen2.QGen0Pu) / Gen2.u0Pu), u0Pu = Complex(1, 0))  annotation(
+    Placement(visible = true, transformation(origin = {-26, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+equation
+//Switch-off equations inhibitions
+  load.switchOffSignal1.value = false;
+  load.switchOffSignal2.value = false;
+  loadPQ.switchOffSignal1.value = false;
+  loadPQ.switchOffSignal2.value = false;
+  line1.switchOffSignal1.value = false;
+  line1.switchOffSignal2.value = false;
+  line2.switchOffSignal1.value = false;
+  line2.switchOffSignal2.value = false;
+  line3.switchOffSignal1.value = false;
+  line3.switchOffSignal2.value = false;
+
+  Gen2.switchOffSignal1.value = false;
+  Gen2.switchOffSignal2.value = false;
+  Gen2.switchOffSignal3.value = false;
+// No variations in the loads
+  der(load.PRefPu) = 0;
+  der(load.QRefPu) = 0;
+  load.deltaP = 0;
+  load.deltaQ = 0;
+  der(loadPQ.PRefPu) = 0;
+  der(loadPQ.QRefPu) = 0;
+  loadPQ.deltaP = 0;
+  loadPQ.deltaQ = 0;
+
+  connect(busIG1.terminal, loadPQ.terminal) annotation(
+    Line(points = {{-2, 40}, {-20, 40}, {-20, 18}}, color = {0, 0, 255}));
+  connect(load.terminal, busL.terminal) annotation(
+    Line(points = {{84, 0}, {72, 0}}, color = {0, 0, 255}));
+  connect(bus.terminal, line1.terminal2) annotation(
+    Line(points = {{16, 0}, {16, 14}}, color = {0, 0, 255}));
+  connect(line1.terminal1, busIG1.terminal) annotation(
+    Line(points = {{16, 34}, {16, 40}, {-2, 40}}, color = {0, 0, 255}));
+  connect(line2.terminal1, bus.terminal) annotation(
+    Line(points = {{16, -36}, {16, 0}}, color = {0, 0, 255}));
+  connect(busIG2.terminal, line2.terminal2) annotation(
+    Line(points = {{-2, -40}, {16, -40}, {16, -16}}, color = {0, 0, 255}));
+  connect(line3.terminal1, busL.terminal) annotation(
+    Line(points = {{60, 0}, {72, 0}}, color = {0, 0, 255}));
+  connect(line3.terminal2, bus.terminal) annotation(
+    Line(points = {{40, 0}, {16, 0}}, color = {0, 0, 255}));
+  connect(slackBus.terminal, busIG1.terminal) annotation(
+    Line(points = {{-36, 40}, {-2, 40}}, color = {0, 0, 255}));
+  connect(Gen2.terminal, busIG2.terminal) annotation(
+    Line(points = {{-26, -40}, {-2, -40}}, color = {0, 0, 255}));
+
+annotation(preferredView = "diagram",
```

Comment:
could you remove : experiment(StartTime = 0, StopTime = 1, Tolerance = 1e-6, Interval = 0.002) from annotation if no reference file is given?